services/director-v2/src/simcore_service_director_v2/models/schemas/services.py

```diff
@@ -57,7 +58,8 @@ class Config:
 
 class ServiceExtras(BaseModel):
     node_requirements: NodeRequirements
-    service_build_details: Optional[ServiceBuildDetails]
+    service_build_details: Optional[ServiceBuildDetails] = None
```

Comment:
hmm, I guess I'll need to review all this Options... this must come from lack of knowledge of pydantic at the time. cause this is probably only optional for old services

Reply:
and that code is probably super old and when we did not know half of what pydantic can do. and that we have services that did not have the build details...